test/matrix/test_matrix.rb

```diff
@@ -583,6 +583,13 @@ def o.coerce(x)
     assert_equal(1, s1 ** o)
   end
 
+  def test_scalar_abs
+    s1 = @a3.abs
+    s2 = @b3.abs
+    assert_equal(s1, Matrix[[4, 1, 3], [0, 3, 7], [11, 4, 2]])
+    assert_equal(s2, Matrix[[7, 7, 10], [9, 3, 2], [1, 3, 9]])
```

Comment:
This second `assert_equal` adds nothing to the first one, could you remove it?

Reply:
Thanks, you're right. Also the name of the test is `test_scalar_abs`, because all previous tests starts with `test_scalar_`. Could it be only `test_abs`?

Reply:
Indeed, `test_abs` is :+1: